spec/configs/stanford_organizations_to_sparql_statements_spec.rb

```diff
@@ -0,0 +1,322 @@
+# frozen_string_literal: true
+
+require 'rdf'
```

Comment:
There's probably no reason to require these in a spec file, because you want to test that your `lib/` code is requiring the dependencies.

Reply:
OK